Tests/Guesser/FilterTypeGuesserTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Sonata\DoctrineORMAdminBundle\Tests\Guesser;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\DoctrineORMAdminBundle\Guesser\FilterTypeGuesser;
+
+class FilterTypeGuesserTest extends TestCase
+{
+    private $guesser;
+    private $modelManager;
+
+    protected function setUp()
+    {
+        $this->guesser = new FilterTypeGuesser();
+        $this->modelManager = $this->prophesize('Sonata\DoctrineORMAdminBundle\Model\ModelManager');
+        $this->metadata = $this->prophesize('Doctrine\ORM\Mapping\ClassMetadata');
+    }
+
+    /**
+     * @expectException Sonata\DoctrineORMAdminBundle\Model\MissingPropertyMetadataException
```

Comment:
The issue is that `expectException()` is not available with old phpunit versions. IIRC we solved this issue already.